### PR TITLE
OSSM-6567: MTT update_ns_package  for Traffic test suite

### PR DESCRIPTION
**Pull request by pbajjuri20**

### CODE REVIEW

File: pkg/tests/tasks/traffic/circuit_breaking_test.go [hunk 1]
```diff
@@ -25,35 +25,36 @@ import (
 	"github.com/maistra/maistra-test-tool/pkg/app"
 	"github.com/maistra/maistra-test-tool/pkg/tests/ossm"
 	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/ns"
 	"github.com/maistra/maistra-test-tool/pkg/util/oc"
 	"github.com/maistra/maistra-test-tool/pkg/util/pod"
 	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
+
 	. "github.com/maistra/maistra-test-tool/pkg/util/test"
```
[the remark this answers is not in the document]
Comment (pbajjuri20) [reply pkg/tests/tasks/traffic/circuit_breaking_test.go:34]: Previously, There were 2 test imports. I try to delete one. but still, it is getting automatically. Will connect and check it.

[the remark this answers is not in the document]
Comment (unsortedhashsets) [reply pkg/tests/tasks/traffic/circuit_breaking_test.go:34]: Do not worry, I will take a look later!

**Review by unsortedhashsets — APPROVED** [review]
LGTM

**Pull request merged**